openapi.yml

```diff
@@ -5569,6 +5730,261 @@ components:
+                id: 1
+                page_type: form_page
+                custom_fields:
+                    -
+                        field_name: question_1
+                        field_value: Red Wine
+                    -
+                        field_name: question_2
+                        field_value: White Wine
+        CustomField:
+            title: Root Type for CustomField
+            description: The root of the CustomField type's schema.
+            required:
+                - field_value
+                - field_name
+            type: object
+            properties:
+                format:
+                    description: The format type of the field
+                    enum:
+                        - string
+                field_name:
+                    description: What the field should be displayed/labeled as
+                    type: string
+                field_value:
+                    description: The value to be displayed for the field
+                    type: string
+            example:
+                format: {}
+                field_name: some text
+                field_value: some text
+        FlexField:
+            title: Root Type for FlexField
+            description: A custom field
+            type: object
+            properties:
+                field_name:
+                    description: Flex field key
+                    type: string
+                field_value:
+                    description: Flex field value
+                    type: string
+            example:
+                field_name: question_1
+                field_value: Red Wine
+        Visitor:
+            title: Root Type for Visitor
+            description: Visitor information
+            type: object
+            properties:
+                id:
+                    format: int32
+                    type: integer
+                active:
+                    type: boolean
+                company:
+                    type: string
+                created_via:
+                    type: string
+                email:
+                    type: string
+                first_name:
+                    type: string
+                last_name:
+                    type: string
+                mobile:
+                    type: string
+                note:
+                    type: string
+                profile_pic_content_type:
+                    type: string
+                profile_pic_file_name:
+                    type: string
+                profile_pic_file_size:
+                    type: string
+                profile_pic_updated_at:
+                    type: string
+                watchlist_level:
+                    type: string
+                created_at:
+                    format: date-time
+                    type: string
+                updated_at:
+                    format: date-time
+                    type: string
+                account_id:
```

Comment:
This isn't for creating is it?
All *real* relationships should reference the actual model and not use the `_id` attributes.